### PR TITLE
Add webservices layer id enhancement

### LINKED ISSUE #348: Add better user experience for users in "web services" modal
Currently we just display the url to the web services. We have had several reports about the lack of usefulness of this. Proposed adding WxS specific parameters to modal.

### PR DESCRIPTION
**Pull request by mejackreed**
This commit also adds the pattern to start removing hard coded Solr field values from the view using Document Presenter pattern

Closes #348

### CODE REVIEW

File: lib/generators/geoblacklight/templates/catalog_controller.rb [hunk 1]
```diff
@@ -33,7 +33,9 @@ class CatalogController < ApplicationController
 
     config.show.display_type_field = 'format'
 
-
+    # Custom GeoBlacklight fields which currently map to GeoBlacklight-Schema
+    # v0.3.2
+    config.wxs_identifier_field = 'layer_id_s'
```
Comment (eliotjordan) [inline lib/generators/geoblacklight/templates/catalog_controller.rb]: What are the advantages to making the wxs_identifier_field configurable? How would this be used in the wild?

Comment (mejackreed) [inline lib/generators/geoblacklight/templates/catalog_controller.rb]: For people who don't want to use GeoBlacklight-Schema, or if GeoBlacklight-Schema changes.

**Comment by eliotjordan** [comment]
The presenter pattern in action! Taking business logic out of views seems much cleaner. This looks good. Just one question (see notes) and this PR is ready to sail.

File: lib/generators/geoblacklight/templates/catalog_controller.rb (hunk 1 above)
Comment (eliotjordan) [inline lib/generators/geoblacklight/templates/catalog_controller.rb]: That makes sense. Especially for schema changes.

**Comment by mejackreed** [comment]
:bow:

**Pull request merged**